libdnf5/repo/repo_downloader.cpp

```diff
@@ -493,6 +513,28 @@ void RepoDownloader::common_handle_setup(LibrepoHandle & h) {
     if (optional_metadata.extract(libdnf5::METADATA_TYPE_UPDATEINFO)) {
         dlist.push_back(MD_FILENAME_UPDATEINFO);
     }
+    if (get_optional_metadata().extract(libdnf5::METADATA_TYPE_APPSTREAM)) {
```

Comment:
This block should be conditional with `#ifdef WITH_APPSTREAM` right?

Reply:
Right, it can be conditional.